### ISSUE TITLE
[bug]wireless energy transfer

### ISSUE THREAD

**Issue by bomdia** [Electrical-Age/ElectricalAge#137]
when rain the energy served fall down ok this is correct but if rain and the rain is blocked by some block the energy fall and this is a bug :)

**Comment by Dolu1990** [comment]
lol XD true
Thanks for issue ^^

**Comment by pjalm** [comment]
rain should not stop it as it dosn't in real world

**Comment by Baughn** [comment]
You're correct, but that sort of checking would be too expensive with the 1.7 interface. 1.10 might be better, maybe; I'll know in a month or three.

Meanwhile, I suggest using the utility poles.
